frontend/src/components/MainHeaderComponent.vue

```diff
@@ -123,7 +139,7 @@ const adminEmail = "forhvap.cliadmin@gov.bc.ca";
       <logo />
     </a>
     
-    <cds-header-name href="javascript:void 0">
+    <cds-header-name ref="cdsHeaderNameRef" href="javascript:void 0">
```

Comment:
In modern web development practices, using href="javascript:void(0)" is generally not recommended. The preferred approach is to separate your JavaScript from your HTML and use event listeners or other methods to handle interactions. This is known as unobtrusive JavaScript.

The reasons to avoid javascript:void(0) include:

**Separation of Concerns:**
Keeping your HTML free of inline JavaScript enhances the separation of concerns, making your code cleaner and more maintainable.

**Accessibility:**
Screen readers and other assistive technologies may not handle JavaScript in links well, leading to potential accessibility issues.

**Graceful Degradation:**
Using unobtrusive JavaScript allows your page to gracefully degrade in situations where JavaScript is disabled or not supported.

Reply:
Fixed.